common/cauthdsl/policy.go

```diff
@@ -32,11 +30,11 @@ func NewPolicyProvider(deserializer msp.IdentityDeserializer) policies.Provider
 func (pr *provider) NewPolicy(data []byte) (policies.Policy, proto.Message, error) {
 	sigPolicy := &cb.SignaturePolicyEnvelope{}
 	if err := proto.Unmarshal(data, sigPolicy); err != nil {
-		return nil, nil, fmt.Errorf("Error unmarshalling to SignaturePolicy: %s", err)
+		return nil, nil, errors.Errorf("error unmarshalling to SignaturePolicy: %s", err)
```

Comment:
If we do stick with `github.com/pkg/errors` for now, I think this one should use errors.Wrap() rather than errors.Errorf() as described at https://hyperledger-fabric.readthedocs.io/en/latest/error-handling.html.

Reply:
Good point, let me update and clean up other places.